modules/gapi/src/backends/common/gbackend.hpp

```diff
@@ -58,6 +58,35 @@ namespace gimpl {
     struct Data;
     struct RcDesc;
 
+    struct RMatMediaBGRAdapter final: public cv::RMat::Adapter
```

Comment:
```.diff
-RMatMediaBGRAdapter
+RMatMediaAdapterBGR
```

?

to avoid "BGRA" misread (and 3 channels below)

Also should avoid `RMatMediaBGRAAdapter` case.

/cc @dmatveev

Reply:
`RMatMediaAdapterBGR` +1

Also, should this class be `EXPORT`-ed if (I believe) it should be used in other G-API modules?

Reply:
Fixed